salvo-remote/azure-pipelines/salvo_controller.yml

```diff
@@ -0,0 +1,55 @@
+# Pipeline steps that execute the salvo-remote controller.
+parameters:
+- name: action
+  displayName: "The action to perform, must be one of [preheat, run_current, run_previous, cooldown]."
+  type: string
+  default: ""
+
+steps:
+- ${{ if eq(parameters.action, 'preheat') }}:
+  - script: "envoy-perf/salvo-remote/azure-pipelines/scripts/asg_control.sh preheat"
+    workingDirectory: "$(Build.SourcesDirectory)"
+    displayName: "preheat the salvo-remote ASG"
```

Comment:
-> Preheat.  Can we use the capital letter for all disaplayName fields?

Reply:
Dine in 46404f9e58dbb3b6be84faf207883ea497ee04fc.